chapters_source.R

```diff
@@ -3,7 +3,8 @@ library(tidyverse)
 library(meetupr)
 
 # meetup groups
-api_key <- readRDS("meetup_key.RDS")
```

Comment:
I will add a comment in the code about this. You are right that this not work. But when you deploy the shiny app on http://shinyapps.io, I couldn't find a way to save the key so using RDS was the way to go. I think I know now how to do this. I will open an issue to address this.